dotCMS/src/main/java/com/dotcms/cluster/common/ClusterServerActionThread.java

```diff
@@ -56,6 +63,24 @@ public void run() {
 					}
 					connection.commit();
 				}
+				    
+				if(LicenseUtil.getLevel()==100 ){
+				    if(noLicense==0){
+				        noLicense=System.currentTimeMillis();
+				    }
+				    //no license for a minute, that is crazy!
+				    if(System.currentTimeMillis() - noLicense > 1000*Config.getLongProperty("CLUSTER_AUTO_RELICENSE_DELAY_SECONDS", 60)){
+				        noLicense=0;
+				        LicenseManager.getInstance().takeLicenseFromRepoIfNeeded();
+				    }
+				}else{
+				    noLicense=0;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)